src/components/searchSection/catergrayList.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { MDBIcon } from 'mdbreact';
+import './myStyle.scss';
+
+function CatergrayList({ items }) {
+  let itemsTags = [];
+  let numberOfItems = items.length;
+  let [ActiveBubble, setActiveBubble] = useState(0);
+  let [setCatogray] = useState('');
+
+  //light the bubble by pink color
+  const handelBubble = (index) => {
+    if (ActiveBubble === index) {
+      return 'textPink';
+    }
+  };
+
+  const emptyItems = () => {
+    for (let i = 1; i <= 9 - numberOfItems; i++) {
+      itemsTags.push(<div md="12" className="lightBlack h-10 mb-1"></div>);
+    }
+  };
+  const getItemList = (e) => {
+    if (e.target !== undefined) {
+      setCatogray(e.target.children[1].innerHTML);
+    }
+  };
```

Comment:
this one also, what dose this method do?

Reply:
get the name of the catogray when the user click on for example popular ,will get popular to display the books related by this type

Reply:
but there is no category variable to use it to render the books

Reply:
also the name of the function is not that clear
and the if statement here, is it important
if it's like that try to refactor the variable 
```
const doseHaveTarget = Boolean(e.target)
if (doseHaveTarget){
...
```